python/xorbits/deploy/kubernetes/config.py

```diff
@@ -502,7 +503,9 @@ def __init__(
         self._startup_probe = startup_probe
 
         self._pre_stop_command = pre_stop_command
+        self._external_storage = external_storage
         self._use_local_image = kwargs.pop("use_local_image", False)
+        self._external_storage_config = kwargs.pop("external_storage_config", dict())
```

Comment:
Should this ``external_storage_config`` just be a valid parameter in new_cluster?

Reply:
Change interface comments accordingly.